src/libs/permission.ts

```diff
@@ -0,0 +1,36 @@
+const permissions = {
+    'getUsers': {
+        all: ['head-trainer'],
+        read: ['trainee', 'trainer'],
+        write: ['trainer'],
+        delete: [],
+    }
+};
+
+
+export default function hasPermission(moduleName, role, permissionType) {
+
+    let type;
+    console.log('Module Name is', moduleName);
+    console.log('permission type is', permissionType);
+  //  console.log('role is', role);
+
+    const values = ['all', 'read', 'write', 'Delete'];
+
+    function getType(permissiontype) {
+        if (values.includes(permissiontype)) { type = permissiontype; }
+        return type;
+    }
+    type = getType(permissionType);
+
+    if (role === 'head-trainer') {
+
+        return true;
+    }
```

Comment:
Do not hardcode the `head-trainer` here check for key `all`. If you need to add one more role with all permission you need to update this code each time.
Fix this